OWML.ModLoader/ModData.cs

```diff
@@ -22,14 +22,14 @@ public class ModData : IModData
                                  Config != null && Config.RequireVR ||
                                  Config == null && DefaultConfig != null && DefaultConfig.RequireVR;
 
-        private IModConfig _configSnapshot;
+        private readonly IModConfig _configSnapshot;
```

Comment:
won't really work if I would use Config.Copy in UpdatedSnapshot

Reply:
ok, so like this? (pushed)